stl/inc/stacktrace

```diff
@@ -278,8 +252,7 @@ public:
 #endif // ^^^ !__cpp_lib_concepts ^^^
     }
 
-    void swap(basic_stacktrace& _Other) noexcept(allocator_traits<_Alloc>::propagate_on_container_swap::value
-                                                 || allocator_traits<_Alloc>::is_always_equal::value) {
+    void swap(basic_stacktrace& _Other) noexcept /* strengthened */ {

```

Comment:
Should I add static_assert about noexcept-ness of `_Frames.swap(_Other._Frames))`, or do the enhancement in the way that I do with `_Noex_move`?